src/pages/community-champions.js

```diff
@@ -122,6 +142,71 @@ const WhatItMeansToBeAChampionText = ({
     );
 };
 
+const LeadByExampleContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkThree};
+    border-radius: ${size.xsmall};
+    padding: ${LEAD_BY_EXAMPLE_CONTAINER_PADDING};
+    margin: ${WHAT_IT_MEANS_TO_BE_A_CHAMPION_ROW_GAP} 0 ${size.xlarge} 0;
+    @media ${screenSize.upToLarge} {
+        padding: ${size.mediumLarge};
+        margin: ${size.large} 0;
+    }
+`;
+
+const LeadByExampleItemContainer = styled('div')`
+    display: grid;
+    grid-column-gap: ${size.mediumLarge};
+    grid-template-columns: ${LEAD_BY_EXAMPLE_ICON_WIDTH} auto;
+    @media ${screenSize.upToLarge} {
+        grid-template-columns: ${LEAD_BY_EXAMPLE_ICON_MOBILE_WIDTH} auto;
+    }
+`;
+
+const LeadByExampleSectionTitle = styled('div')`
+    font-family: 'Fira Mono', monospace;
+    font-weight: bold;
+    margin-bottom: ${size.xsmall};
+    font-size: ${fontSize.small};
+    line-height: ${lineHeight.small};
+    @media ${screenSize.upToLarge} {
+        font-size: ${fontSize.tiny};
+        line-height: ${lineHeight.tiny};
+    }
+`;
+
+const LeadByExampleItem = ({ icon, title, description }) => {
+    return (
+        <LeadByExampleItemContainer>
+            <img src={icon} />
+            <div>
+                <LeadByExampleSectionTitle>{title}</LeadByExampleSectionTitle>
+                {description}
+            </div>
+        </LeadByExampleItemContainer>
+    );
+};
```

Comment:
```suggestion
const LeadByExampleItem = ({ icon, title, description }) => (
        <LeadByExampleItemContainer>
            <img src={icon} />
            <div>
                <LeadByExampleSectionTitle>{title}</LeadByExampleSectionTitle>
                {description}
            </div>
        </LeadByExampleItemContainer>
    );
```

We could use the shorthand syntax since we are just returning here!